runtime/src/main/java/com/datastax/oss/quarkus/runtime/api/reactive/MutinyReactiveResultSet.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.api.reactive;
+
+import com.datastax.dse.driver.api.core.cql.reactive.ReactiveResultSet;
+import com.datastax.dse.driver.api.core.cql.reactive.ReactiveRow;
+import io.smallrye.mutiny.Multi;
+
+public interface MutinyReactiveResultSet
```

Comment:
Still missing some minimal class javadocs for this interface and other similar ones.

Reply:
done in: https://github.com/datastax/cassandra-quarkus/pull/11/commits/8eec3bf295740eab850665c67a02c9ad8960efea
Feel free to re-arrange those